PKGBUILD

```diff
@@ -12,7 +12,6 @@ _build_opt=1
 
 pkgname=()
 [ "$_build_no_opt" -eq 1 ] && pkgname+=(tensorflow-rocm python-tensorflow-rocm)
-[ "$_build_opt" -eq 1 ] && pkgname+=(tensorflow-opt-rocm python-tensorflow-opt-rocm)
```

Comment:
Why is this line deleted? Can you add it back?

Reply:
I took it out because the compile took very long and i didn't figure out how to inhibit the opt version from building. Am i missing something?

Reply:
Yes the PKGBUILD should build both packages, when testing you can build just one. For an easy bypass just change the variable above to 0 that you do not want. i.e. _build_opt=0 on line 11.